src/Appwrite/Platform/Workers/Functions.php

```diff
@@ -83,6 +83,7 @@ public function action(Message $message, Database $dbForProject, Func $queueForF
         $eventData = $payload['payload'] ?? '';
         $project = new Document($payload['project'] ?? []);
         $function = new Document($payload['function'] ?? []);
+        $functionId = $payload['functionId'] ?? '';
```

Comment:
Related to previous comment, we can `$function->getId()` I think

Reply:
Please see above.